index.html

```diff
@@ -18,20 +19,142 @@
     </nav>
   </header>
   <main>
-    <section class="1 intro-text">
-      <h1 class="primary-text">Hey There. I'm Carlos</h1>
-      <h3 class="supporting-text">I'm a Software Developer</h3>
-      <p class="Description-text">I can help you build a product , feature or website Look through some of my work<br> and experience! If you like what you see<br> and have a project you need coded,<br> 
-          don’t hestiate to contact me.</p>
+    <section class="head-section">
+      <section class="1 intro-text">
+        <h2 class="primary-text">Hey There. I'm Carlos</h2>
+        <h4 class="supporting-text">I'm a Software Developer</h4>
+        <p class="Description-text">I can help you build a product , feature or website Look through some of my work<br> and experience! If you like what you see<br> and have a project you need coded,<br> 
+            don’t hestiate to contact me.</p>
+      </section>
+      <section>
+        <ul class="2 social-media">
+          <li class="icon github"><img src="assets/github.svg" alt="github.com icon"></li>
+          <li class="icon linkedin"><img src="assets/Linkedin-icon.svg" alt="linkedin.com icon"></li>
+          <li class="icon angel-list"><img src="assets/angel-list.svg" alt="angel icon"></li>
+          <li class="icon twitter"><img src="assets/twitter.svg" alt="twitter.com icon"></li>
+          <li class="icon medium"><img src="assets/medium.svg" alt="medium.com icon"></li>
+        </ul>
+      </section>
     </section>
-    <section>
-      <ul class="2 social-media">
-        <li class="icon github"><img src="assets/github.svg" alt="github.com icon"></li>
-        <li class="icon linkedin"><img src="assets/Linkedin-icon.svg" alt="linkedin.com icon"></li>
-        <li class="icon angel-list"><img src="assets/angel-list.svg" alt="angel icon"></li>
-        <li class="icon twitter"><img src="assets/twitter.svg" alt="twitter.com icon"></li>
-        <li class="icon medium"><img src="assets/medium.svg" alt="medium.com icon"></li>
-      </ul>
+    <section class="works-section">
+        <div class="title-works-section">
+          <h2>My Recent Works</h2>
+        </div>
+        <section class="card principal-card">
+          <div class="feat-img-principal-card"></div>
+          <div class="description-principal-card">
+            <h3>Multi-Post Stories</h3>
+            <p class="supporting-text-principal-card">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <div class="tag-holder">
```

Comment:
- The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.

Reply:
## Done!